iai_apartment/urdf/kitchen.xarco

```diff
@@ -1649,4 +1649,94 @@
 		</link>
 
 	</xacro:macro>
+
+<!-- extras -->
+	<!-- Collision okay surface for cabinet1-->
```

Comment:
Is the indentation off in this file?

Reply:
Nope. Readability. The indentations don't make any difference in XML. It's a usability and maintainability feature.